snuba/datasets/processors/replays_processor.py

```diff
@@ -273,3 +283,35 @@ def stringify(value: Any) -> str:
         return ""
     else:
         return str(value).encode("utf8", errors="backslashreplace").decode("utf8")
+
+
+def _intify(value: Any) -> int:
+    """Return an integer or err."""
+    if isinstance(value, int):
+        return value
+    elif isinstance(value, (str, float)):
+        return int(value)
+    else:
+        raise TypeError(
+            f'Expected value of type "int"; received "{type(value)}" with a value of "{value}".'
+        )
+
+
+def _collapse_or_err(callable: Callable[[int], int | None], value: int) -> int:
+    """Return the integer or error if it overflows."""
+    if callable(value) is None:
+        # This exception can only be triggered through abuse.  We choose not to suppress these
+        # exceptions in favor of identifying the origin.
+        raise ValueError(f'Integer "{value}" overflowed.')
```

Comment:
Won't this cause the whole replay processing pipeline to stop if one org is causing the abuse?

Reply:
This particular error *should* trigger the replays DLQ (let me know if this assumption is incorrect).  An exception raised after the processing step (e.g. the write step) caused INC-250.  I don't believe this change introduces any pipeline risk.